### DISCUSSION TITLE
Changing model output to texts only

### DISCUSSION THREAD

**Discussion by hansoullee20** [ultralytics/yolov5#5615] [Q&A]
Hello

I am wondering if there are ways to change the model to always output the resulting anchor boxes in a text format.
My project involves training the model and export the model in .onnx format and eventually convert it to .dlc format.
This is because the ultimate goal is to run the trained yolov5 model in snpe(Snapdragon Neural Processing Engine) environment using .dlc format and output the anchor box in text form.

I have been looking into the source codes to understand the model structure, but have yet to found the right functions to modify.
How can I change the model to output the anchor boxes in a text format?

Thank you in advance

**Comment by glenn-jocher** [comment]
@hansoullee20 I'm sorry I don't understand the question. In python you can convert integers or floats to strings easily:
```python
str(123)
Out[5]: '123'
```
